currencyii/src/main/java/nl/tudelft/trustchain/currencyii/ui/bitcoin/JoinDAOFragment.kt

```diff
@@ -72,11 +72,19 @@ class JoinDAOFragment() : BaseFragment(R.layout.fragment_join_network) {
     private fun fetchSharedWalletsAndUpdateUI() {
         lifecycleScope.launch {
             withContext(Dispatchers.IO) {
+                alert_tf?.text = "Crawling blocks for DAOs..."
+
                 val discoveredWallets = getCoinCommunity().discoverSharedWallets()
                 updateSharedWallets(discoveredWallets)
                 updateSharedWalletsUI()
                 crawlAvailableSharedWallets()
                 updateSharedWalletsUI()
+
+                if (fetchedWallets.isEmpty()) {
+                    alert_tf?.text = "No DAOs found."
+                } else {
```

Comment:
Can you set the text to View.GONE so that it does not take any space?

Reply:
We also do that in the MyDAOs screen.

Reply:
That would also mean that you need to make sure to change the visibility to `View.VISIBLE`, is that also done in those other files or not?

Reply:
Could make a method that sets the alert and sets the visibility to `View.VISIBLE`

Reply:
Should be fixed now in https://github.com/InvictusRMC/trustchain-superapp/pull/19/commits/cc504a9c3b38895ce8985f7a646ad02ef65021bb